osquery/tables/system/linux/pci_devices.cpp

```diff
@@ -392,10 +393,20 @@ QueryData genPCIDevices(QueryContext& context) {
     return results;
   }
 
-  std::ifstream raw(kPciidsPath);
+  std::ifstream raw;
+  std::string all_path;
+  for (const std::string& kPciidsPath : kPciidsPathList) {
+    raw.open(kPciidsPath);
+    if (raw) {
+      break;
+    }
+    if (!all_path.empty())
+      all_path.append(" ");
```

Comment:
Is `all_path` used to display a friendly error?

Instead just look for a valid path and if you cannot find any then construct a string using `osquery::join` for the error message.